core/shared/src/main/scala/scalaz/zio/ZSchedule.scala

```diff
@@ -790,9 +834,33 @@ object ZSchedule extends Schedule_Functions {
                 val n = 1 +
                   (if (await < 0) ((now - start) / nanos).toInt else n0)
 
-                Decision.cont(Duration.fromNanos(await.max(0L)), (start, n, i + 1), i + 1)
+                Decision.cont(Duration.fromNanos(await.max(0L)).relative, (start, n, i + 1), i + 1)
               }
           }
       )
   }
+
+  /**
+   * Builds an Schedule capable of running an effect at a given minute and hour, every day
+   */
+  final def everyDayAt(minute: Int, hour: Int): ZSchedule[Clock, Unit, (Long, Long)] =
+    ZSchedule[Clock, Long, Unit, (Long, Long)](
+      initial0 = clock.currentTime(unit = TimeUnit.MILLISECONDS).map(_ => 0L),
+      update0 = (_, timesRan) =>
```

Comment:
We should at least document the dependency on java.time in the microsite, on the Scala.js page.